### PR TITLE
Improve vSphere integration guide

### PR DESCRIPTION
**Pull request by prees1**
Improves the content in the vSphere integration guide.

- Corrects some content and steps that were out of date.
- Adds more information for the needed inputs and values.
- Improves the wording on the existing content to make it easier to read and follow.
- Reorders some content, moving the 'other' configuration options to the end. They are optionl, not really need used by most people.
- Also improves the markdown spacing and formatting, no visible impact on render though.

### CODE REVIEW

**Comment by jmariomejiap** [comment]
I think this a great improvement, it flows is better ❤️,

File: pages/tidal tools/vsphere.md [hunk 1]
```diff
@@ -6,62 +6,87 @@ sidebar: main_sidebar
...
+Look at [setting this command up as a cron job](https://www.digitalocean.com/community/tutorials/how-to-use-cron-to-automate-tasks-on-a-vps) that once per day.
 
-``` tidal config set vsphere.password [your password] ```
+## Additional Configuration Options
+The integration can be configured with several other methods besides `tidal login`.
+You can use these alternate approaches if they better suite your use case.
 
-``` tidal config set vsphere.server 192.168.1.12 ```
+The other options are to use, the `tidal config` command, a config file via `--config` or via environment variables.
+If a config file is present it will be used, if it is not present environment variables will be used.
```
Comment (justinbarclay) [inline pages/tidal tools/vsphere.md:76]: I think AWS cli sets a good example of (how to do this)[https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-precedence]

The Tidal Tools credentials and configuration settings take precedence in the following order:
1. Configuration file
2. Environment Variables
3. Prompt in the terminal

**Review by justinbarclay — COMMENTED** [review]
Some alternate takes and fixes for typos :)

File: pages/tidal tools/vsphere.md [hunk 2]
```diff
@@ -6,62 +6,87 @@ sidebar: main_sidebar
...
 folder: tidaltools
 ---
-
-
-
 ## Getting Started
-To get started you will need to have Tidal Tools installed. You can download a copy from https://get.tidal.sh and check out this [article](tidal-tools.html) on how to install.
-
-
+To get started you will need make sure you have a copy of [Tidal Tools](https://get.tidal.sh) and it is [installed](tidal-tools.html).
```
[the remark this answers is not in the document]
Comment (prees1) [reply pages/tidal tools/vsphere.md]: 👍 left the longer sentence form link off to for the sake of being brief.

File: pages/tidal tools/vsphere.md (hunk 1 above)
Comment (prees1) [reply pages/tidal tools/vsphere.md]: Good idea. I followed that pattern here too. I took it a bit farther and added some more to make it clear too. Also, I think the precedence is '1. env var / 2. config file', so I listed it that way.

What do you think of how this reads now?

File: pages/tidal tools/vsphere.md [hunk 3]
```diff
@@ -6,62 +6,87 @@ sidebar: main_sidebar
...
 
-However, if you are already logged on to the Tidal API, you can skip to execute the `tidal login vsphere` command, which will _only_ prompt you to
-login to vSphere.
+A set of prompts will appear to help you connect to your vSphere account:
 
-If you do wish to login to your vSphere account, you can select the option `y`:
+1. vSphere Server - Either an IP address or a FQDN that resolves to the correct IP for the ESXi or vCenter instance
+2. vSphere Username - A username that has read access to the vCenter or ESXi instance.
+3. vSphere Password - Corresponding password for the user.
+4. vSphere API Endpoint - The default is `/sdk`, unless you know it is not this default, it can left as is.
```
[the remark this answers is not in the document]
Comment (prees1) [reply pages/tidal tools/vsphere.md]: hm ok, I was trying to help explain _when necessary_ is exactly. And that often times it is _not_ necessary.

What do you think of it now?

File: pages/tidal tools/vsphere.md [hunk 4]
```diff
@@ -6,62 +6,87 @@ sidebar: main_sidebar
...
+5. Use SSL/TLS? - The default is to use SSL (y), which can be left unless you know you can't use it.
```
[the remark this answers is not in the document]
Comment (prees1) [reply pages/tidal tools/vsphere.md:28]: I'm not following your suggestion here

File: pages/tidal tools/vsphere.md [hunk 5]
```diff
@@ -6,62 +6,87 @@ sidebar: main_sidebar
...
+You can easily set this to run periodically. The integration updates records if they already exist, or creates new records if they don't.
+Look at [setting this command up as a cron job](https://www.digitalocean.com/community/tutorials/how-to-use-cron-to-automate-tasks-on-a-vps) that once per day.
 
-``` tidal config set vsphere.password [your password] ```
+## Additional Configuration Options
+The integration can be configured with several other methods besides `tidal login`.
+You can use these alternate approaches if they better suite your use case.
 
-``` tidal config set vsphere.server 192.168.1.12 ```
+The other options are to use, the `tidal config` command, a config file via `--config` or via environment variables.
```
[the remark this answers is not in the document]
Comment (prees1) [reply pages/tidal tools/vsphere.md:75]: How about now? I simplified it a bit, I think

File: pages/tidal tools/vsphere.md [hunk 6]
```diff
@@ -73,43 +98,13 @@ tidal:
   password: my_secure_password_here
   url: https://my_instance_name_here.tidalmg.com
 ```
+Could be used with a command: `tidal sync vsphere --config config.yml`
 
 ### 3) Environment Variables
-
-If you would like to set these credentials and settings via environment variables, you can use these variables:
+To configure with environment variables, you can use these variables to set the values and Tidal Tools will use them.
```
[the remark this answers is not in the document]
Comment (prees1) [reply pages/tidal tools/vsphere.md]: Yea it was unnecessarily wordy. How about now?

File: pages/tidal tools/vsphere.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmariomejiap) [reply pages/tidal tools/vsphere.md]: yeah, I see your point. maybe is the **unless** word.. I need more ruby in my life. thanks for explaining 👍

[the remark this answers is not in the document]
Comment (prees1) [reply pages/tidal tools/vsphere.md]: Yea I agree it was worded poorly. As a heads up github is showing the `Outdated` diff here. I did try and simplify the wording in a newer commit.

File: pages/tidal tools/vsphere.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jmariomejiap) [reply pages/tidal tools/vsphere.md:28]: It sounded better in my head yesterday. Im not sure now 😑. please ignore.

File: pages/tidal tools/vsphere.md (hunk 1 above)
Comment (justinbarclay) [reply pages/tidal tools/vsphere.md]: Yeah it reads much nice now! Yeah, I was just following the order that you had in the sentences - so it was meant more as a rough idea than something to adhere to.

File: pages/tidal tools/vsphere.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jmariomejiap) [reply pages/tidal tools/vsphere.md]: cool, It sounds better, thanks 👍

**Review by jmariomejiap — COMMENTED** [review]
🚀
